administration/scheduling-and-retries.md

```diff
@@ -10,6 +10,41 @@ Once an output plugin gets called to flush some data, after processing that data
 
 If the return status was **OK**, it means it was successfully able to process and flush the data, if it returned an **Error** status, means that an unrecoverable error happened and the engine should not try to flush that data again. If a **Retry** was requested, the _Engine_ will ask the _Scheduler_ to retry to flush that data, the Scheduler will decide how many seconds to wait before that happens.
 
+## Configuring Wait Time for Retry
+
+The Scheduler provides two configuration options called **scheduler.cap** and **scheduler.base** which can be set on service section. 
+
+| Key | Description | Default Value | 
+| -- | ------------| --------------| 
+| scheduler.cap | Set a maximum retry time in second. The property is supported from v1.8.7. | 2000 | 
+| scheduler.base | Set a base of exponential backoff. The property is supported from v1.8.7. | 5 |
```

Comment:
These are also mentioned in other places: 
- https://github.com/fluent/fluent-bit-docs/blob/master/administration/configuring-fluent-bit/classic-mode/configuration-file.md
- https://github.com/fluent/fluent-bit-docs/blob/master/administration/configuring-fluent-bit/yaml/configuration-file.md

@DrewZhang13 I think we need to make sure that any place the user finds these settings, they are directed to this description.  So I think we need a link or update there too.

Reply:
Good point, will add links in those places for reference